relay-system/Cargo.toml

```diff
@@ -17,7 +17,7 @@ futures = { workspace = true }
 once_cell = { workspace = true }
 relay-log = { workspace = true }
 relay-statsd = { workspace = true }
-tokio = { workspace = true, features = ["rt", "signal", "macros", "sync", "time"] }
+tokio = { workspace = true, features = ["rt", "signal", "macros", "sync", "time", "rt-multi-thread"] }
```

Comment:
This does not actually change the runtime right? We were already using the multi-threaded version?

Reply:
Yes, we just now create the runtime in the `system` crate where was before we did it in the server crate which has the feature.